ws/websocket.go

```diff
@@ -973,7 +985,8 @@ func (client *Client) cleanup() {
 
 func (client *Client) handleReconnection() {
 	log.Info("started automatic reconnection handler")
-	delay := client.timeoutConfig.ReconnectBackoff
+	delay := client.timeoutConfig.RetryBackOffWaitMinimum + time.Duration(rand.Intn(client.timeoutConfig.RetryBackOffRandomRange+1))
```

Comment:
First of all thanks a lot for the contribution. I agree with you that the incremental backoff with range can be used for v1.6 as well, since nothing really speaks against it.

Regarding the implementation, I see the following issues:
1. the back-off random range is being added only once
2. the range is being added as nanoseconds, which doesn't really add much randomness -> I presume this should be cast to seconds

### Design suggestion

Just to quote the specs:
> When the Charging Station is reconnecting, after a connection loss, it will use this variable as the maximum value for the random part of the back-off time. It will add a new random value to every increasing back-off time, including the first connection attempt (with this maximum), for the amount of times it will double the previous back-off time. When the maximum number of increments is reached, the Charging Station will keep connecting with the same back-off time.

My interpretation is that for every iteration of the reconnect loop, the new delay should be:
```go
delay *= 2
delay += time.Duration(rand.Intn(client.timeoutConfig.RetryBackOffRandomRange+1)) * time.Seconds
```
Does this make sense or did you reach a different interpretation?